static/js/util/validation.js

```diff
@@ -11,12 +11,17 @@ let handleNestedValidation = (profile, keys, nestedKey) => {
   return _.flatten(profile[nestedKey].map((v, i) => nestedFields(i)));
 };
 
+/**
+ * Is the value === undefined, null, or ""?
+ */
```

Comment:
I don't think this comment is super necessary, I think this function documents itself. Personal style preference though.